.travis.yml

```diff
@@ -4,10 +4,9 @@ before_install:
   - npm set progress=false
 language: node_js
 node_js:
+  - "6.0"
```

Comment:
the aim is to support the lts versions https://cloud.githubusercontent.com/assets/1156093/16716136/32f18fa6-46f4-11e6-8d56-3619807d38ad.png